.travis.yml

```diff
@@ -8,7 +8,7 @@ osx_image: xcode6.4
 
 env:
   matrix:
-    
+    - CONDA_R=3.3.1
```

Comment:
@ocefpaf this pretty mich depends on how fast we can move on to R 3.4. If this takes longer we need either to build everything in bioconda against 3.3.2 or here a few packages against 3.3.1.